tests/integration/test_sns.py

```diff
@@ -467,7 +468,7 @@ def test_subscribe_platform_endpoint(
         self, sns_client, sqs_create_queue, sns_create_topic, sns_subscription
     ):
 
-        sns_backend = SNSBackend.get()
+        sns_backend = sns_stores[get_aws_account_id()][aws_stack.get_region()]
```

Comment:
nit: I guess we could simply use `SnsProvider.get_store()` here, which would be slightly shorter

Reply:
Good catch, fixed with 36957feb